enterprise-suite/es-monitor-api/bootstrap-monitors.json

```diff
@@ -608,11 +608,11 @@
       "model": "threshold",
       "parameters": {
         "name": "akka_http_server_5xx",
-        "metric": "akka_http_http_server_responses_5xx",
+        "metric": "akka_http_http_server_responses_5xx_rate",
         "comparator": ">",
         "threshold": "0",
         "window": "5m",
-        "confidence": "1",
+        "confidence": "0.25",
```

Comment:
this makes it slow to resolve, so those annoying errors at the beginning will linger for 5m.  setting to "1" makes it go green as soon as the 5xx errors stop